### PR TITLE
Localytics 3 usertrack

### PR DESCRIPTION
**Pull request by langfors**
This upgrades localytics to v3.0 and adds user attribution.

### CODE REVIEW

**Comment by timelf123** [comment]
Thanks for the contribution! Checkout out my commit comment and let me know your thoughts

File: README.md [hunk 1]
```diff
@@ -35,13 +35,23 @@ angular.module('myApp', [
 ### bower
 
 ```shell
-bower install angulartics-localytics
+bower install gemr/angulartics-localytics
```
Comment (joehalliwell) [inline README.md:38]: Please revert to the official package!

**Comment by joehalliwell** [comment]
Aside from the documentation niggle mentioned above, this looks good to me. Thank you very much for the contribution!

**Comment by langfors** [comment]
we're pretty happy with this now; works well for us!

**Comment by timelf123** [comment]
Thanks!

**Pull request merged**